components/HomepageNews/HomepageNews.vue

```diff
@@ -122,6 +133,30 @@ export default {
   },
   methods: {
     isInternalLink,
+    async fetchBitlyLinks() {
+      this.upcomingNews.forEach(async item => {
+        const url = pathOr("", ['fields', 'url'], item)
+        if (url.includes('bit.ly')) {
+          const bitlyId = url.replace("https://", "")
+          try {
+            const response = await this.$axios.post(process.env.bitly_expand_endpoint, { bitlink_id: bitlyId }, {
+              headers: {
+                Authorization: `Bearer ${process.env.BITLY_ACCESS_TOKEN}`,
+                'Content-Type': 'application/json',
+              }
+            })
+            const newUrl = response.data.long_url
+            this.itemIsInternalLink.push(isInternalLink(newUrl))
+          } catch {
+            console.log("Error retreiving bitly link destination url")
```

Comment:
you might already know this and chose to do this to have a custom error, but just in case you didn't -->  you can use console.error for this is you want and it will print the error message from the API 

```
catch (err) {
   console.error(err) 
}
```
or you could console.log the error too.